packages/showcases/svelte/stories/components/VtmnButton/VtmnButtonView.svelte

```diff
@@ -4,6 +4,8 @@
    * @wrapper
    */
   import VtmnButton from '@vtmn/svelte/src/components/VtmnButton.svelte';
+
+  export let children = 'Button';
 </script>
 
-<VtmnButton {...$$restProps}>Button</VtmnButton>
+<VtmnButton {...$$restProps}>{children}</VtmnButton>
```

Comment:
is it really $$resProps you want or $$props would suffice ? 

https://svelte.dev/docs#Attributes_and_props

Reply:
Yes you're right! Thanks Xavier